.github/workflows/update-manifest.yml

```diff
@@ -111,21 +105,71 @@ jobs:
             yq e -i '.deployment.container.image = "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/next-build-node:${{ env.IMAGE_TAG }}"' values.yaml
             git diff
 
-      - name: Add and Commit file (triggered by upstream)
+      - name: Add and Commit file, Staging Next Build
         if: ${{ github.event_name == 'workflow_run' }}
         uses: EndBug/add-and-commit@a94899bca583c204427a224a7af87c02f9b325d5 # v9.1.4
         with:
           add: '*.yaml'
-          cwd: vsp-infra-application-manifests/apps/${{ matrix.app_name }}/${{ matrix.environment }}
+          cwd: vsp-infra-application-manifests/apps/next-build/staging
+          author_name: va-vsp-bot
+          author_email: devops@va.gov
+          message: 'auto update next-build images and helm chart'
+
+      - name: Update image and helm chart versions, Prod Next Build
+        if: ${{ github.event_name == 'workflow_run' }}
+        run: |
+            cd vsp-infra-application-manifests/apps/next-build/prod
+            yq e -i '.deployment.container.image = "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/next-build-node:${{ env.IMAGE_TAG }}"' values.yaml
+            git diff
+
+      - name: Add and Commit file, Prod Next Build
+        if: ${{ github.event_name == 'workflow_run' }}
+        uses: EndBug/add-and-commit@a94899bca583c204427a224a7af87c02f9b325d5 # v9.1.4
+        with:
+          add: '*.yaml'
+          cwd: vsp-infra-application-manifests/apps/next-build/prod
+          author_name: va-vsp-bot
+          author_email: devops@va.gov
+          message: 'auto update next-build images and helm chart'
+
+      - name: Update image and helm chart versions, Staging Next Build Test
+        if: ${{ github.event_name == 'workflow_run' }}
+        run: |
+            cd vsp-infra-application-manifests/apps/next-build-test/staging
+            yq e -i '.deployment.container.image = "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/next-build-node:${{ env.IMAGE_TAG }}"' values.yaml
+            git diff
+
+      - name: Add and Commit file, Staging Next Build Test
+        if: ${{ github.event_name == 'workflow_run' }}
+        uses: EndBug/add-and-commit@a94899bca583c204427a224a7af87c02f9b325d5 # v9.1.4
+        with:
+          add: '*.yaml'
+          cwd: vsp-infra-application-manifests/apps/next-build-test/staging
+          author_name: va-vsp-bot
+          author_email: devops@va.gov
+          message: 'auto update next-build images and helm chart'
+
+      - name: Update image and helm chart versions, Prod Next Build Test
+        if: ${{ github.event_name == 'workflow_run' }}
+        run: |
+            cd vsp-infra-application-manifests/apps/next-build-test/prod
+            yq e -i '.deployment.container.image = "008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/next-build-node:${{ env.IMAGE_TAG }}"' values.yaml
+            git diff
+
+      - name: Add and Commit file, Prod Next Build Test
```

Comment:
I think this can be simplified to a single add-and-commit at the end here. If the working dir is 'vsp-infra-application-manifests/apps' (or higher) the add for '*.yaml' should recursively add all changed yaml files and it will be one commit.